docs/node-operators/requirements/infrastructure.md

```diff
@@ -24,6 +24,20 @@ The impact of using fewer cores than recommended is that the critical parts (dat
 
 The impact of having slower cores than recommended (or older cores that have a reduced IPC rate compared to modern Zen3 cores) is that the maximum throughput will be reduced due to the bottlenecks being single threaded.
 
+### Storage recommendations 
+Performance testing on NVMe SSD, SATA SSD and traditional HDD (5400rpm) storage devices was done to see how they handled the workload of running a validator node. The testing included running the node for an hour and measuring the number of blocks it replayed, and the service time of the storage device.
```

Comment:
**thought (non-blocking)**:  The important bit should go first - which is *use SSD or NVME*. I had to wade through this whole paragraph about boring benchmarks!